File: bubble_sort.js

bubbleSort = (arr) => {
    let count = arr.length

    while (count > 0) {
        arr.forEach((n, i) => {
            if (arr[i] > arr[i + 1]) {
                let temp = arr[i]
                arr[i] = arr[i + 1]
                arr[i + 1] = temp
            }           
        })
      count--
    }
    // return arr   
    console.log(arr) 
}


bubbleSort([4,3,78,2,0,2])

File: bubble_sort.rb


def bubble_sort(arr)
    index = 0

    while index < arr.length - 1 do
        if arr[index] > arr[index + 1]
            arr[index], arr[index + 1] = arr[index + 1], arr[index]
            index = 0
        else
            index += 1
        end
    end
    arr
end

bubble_sort([7, 3, 1, 5])



